acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/Eth2AltairBlockSigningAcceptanceTest.java

```diff
@@ -77,7 +72,7 @@ void signAndVerifyBlockV2Signature(final SpecMilestone specMilestone) throws Exc
   @Test
   void signAndVerifyLegacyBlockSignature() throws Exception {
     final Eth2BlockSigningRequestUtil util = new Eth2BlockSigningRequestUtil(SpecMilestone.PHASE0);
-    setupSigner("eth2", null, "minimal", Optional.empty());
+    setupSigner("eth2", null, "minimal", null);
```

Comment:
Might be overkill in this case, but could avoid passing the null into the Optional with another overloaded setupSigner method which doesn't call `withAltairForkEpoch`

Reply:
you are correct, its technically only for legacy block signing that we need to start web3signer without altair fork. Rest of signing types works with altair fork.

Reply:
Rather we didn't pass in null as well, even an optional altair block would be nicer. Might be nit picking though.